src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -25,122 +26,31 @@ public DeploymentOptionProperty(T value, string description = "", string display
             this.DisplayName = displayName;
         }
 
-        // Default and selected value of the deployment options
+        /// <summary>
+        /// Default and selected value of the deployment option
+        /// </summary>
         public T Value { get; set; }
 
-        // Description of the deployment options
+        /// <summary>
+        /// Description of the deployment option
+        /// </summary>
         public string Description { get; set; }
 
-        // To display the options in ADS extensions UI in SchemaCompare/SQL-DB-Project/Dacpac extensions
+        /// <summary>
+        /// Display name of the deployment option
+        /// </summary>
         public string DisplayName { get; set; }
     }
 
     /// <summary>
     /// Class to define deployment options.
-    /// Keeping the order and defaults same as DacFx
-    /// The default values here should also match the default values in ADS UX
-    /// NOTE: When new deployment options are added in DacFx, they need to be added here too
+    /// The default property names here should also match the ADS UX
```

Comment:
What do you mean "should match"? Is that defined somewhere else? A comment like this makes me think that there's something that needs to be kept in sync, if that's true then provide a link to that (and in the other one a link to here) otherwise reword this to make it clearer what you mean by this.